libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -225,42 +215,67 @@ private <T> T getJsonProperty(
         }
     }
 
-    private IdentityClaim getIdentityClaim(
-            String credential, String credentialIssuer, boolean validateCorrelation)
-            throws HttpResponseExceptionWithErrorBody {
-        JsonNode vcClaimNode = getVCClaimNode(credential);
+    private IdentityClaim getIdentityClaim(String credential)
+            throws HttpResponseExceptionWithErrorBody, CredentialParseException {
+        JsonNode vcClaimNode = getVCClaimNode(credential, VC_CREDENTIAL_SUBJECT);
         List<Name> names =
                 getJsonProperty(
                         vcClaimNode,
                         NAME_PROPERTY_NAME,
-                        credentialIssuer,
                         objectMapper
                                 .getTypeFactory()
-                                .constructCollectionType(List.class, Name.class),
-                        validateCorrelation);
+                                .constructCollectionType(List.class, Name.class));
         List<BirthDate> birthDates =
                 getJsonProperty(
                         vcClaimNode,
                         BIRTH_DATE_PROPERTY_NAME,
-                        credentialIssuer,
                         objectMapper
                                 .getTypeFactory()
-                                .constructCollectionType(List.class, BirthDate.class),
-                        validateCorrelation);
+                                .constructCollectionType(List.class, BirthDate.class));
 
         return new IdentityClaim(names, birthDates);
     }
 
-    private Optional<IdentityClaim> generateIdentityClaim(List<VcStoreItem> successfulVCStoreItems)
+    public Optional<IdentityClaim> generateIdentityClaim(List<VcStoreItem> successfulVCStoreItems)
```

Comment:
Does this need to be made public? Looks like it's only called from within this class.

Reply:
In fact, do we need it at all? Looks like a wrapper for `findIdentityClaim`, just with a guard clause

Reply:
Put this back to private - this method is doing slightly more now in filtering evidence vcs from its `successfulVcStoreItems` list so I'd say it probably warrants existence for now